migration/migrating-4-4/migrating-openshift-4-to-4.adoc

```diff
@@ -0,0 +1,91 @@
+[id="migrating-openshift-4-to-4"]
+= Migrating {product-title} 4.1 to 4.2
+include::modules/common-attributes.adoc[]
+:context: ocp-4_1-4_2
```

Comment:
Usually we do try to have the context match the anchor of the assembly (`migrating-openshift-4-to-4` in this case)